core/domain/feedback_jobs_one_off_test.py

```diff
@@ -68,6 +68,8 @@ def _run_one_off_job(self):
         self.assertEqual(
             self.count_jobs_in_taskqueue(
                 taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
+        self.assertEqual(
```

Comment:
This does not address my earlier comment. I'll try to explain again:

The point I was making was that you are checking the message count for the state of affairs post-job. You need to do a similar check pre-job to make sure the message counts are all what you expect them to be _before_ the job is run.

Reply:
Hi @seanlip, thanks for picking this issue, I checked by adding the pre-job ```assertEqual``` statements that the tests were not getting passed due to the job, rather it was getting passed due to the get_message_count  which I made in the ```feedback_services.py``` which used the ```gae_models.py``` in ```core/storage/feedback``` and it calculated the message_count value then returned it. Therefore the test was passing. So I have made a new function in ```feedback_services``` which returns the instant value of message_count.